### PR TITLE
[8.0][website_crm_quick_answer] Form quick answer

### PR DESCRIPTION
**Pull request by yajo**
Relaunch https://github.com/OCA/website/pull/86.

### CODE REVIEW

**Comment by yajo** [comment]
I'd thank some help with travis, it fails some phantomjs website_event_sale test, but does not seem related.

**Comment by yajo** [comment]
cc @rafaelbn @sergio-incaser

**Comment by yajo** [comment]
CC @pedrobaeza @carlos-incaser.

**Comment by rafaelbn** [comment]
Tested un runbot :+1:
 cc @dgzurita @janetrp really useful for websites in Odoo

**Comment by janetrp** [comment]
Tested and checked in the sent messages. :+1:

**Comment by rafaelbn** [comment]
This PR functionally looks ready to merge, please @pedrobaeza could you help us with travis?
cc @carlos-incaser @sergio-incaser

**Comment by pedrobaeza** [comment]
Is this branch rebased with latest fixes?

**Comment by JavierIniesta** [comment]
Tested in runbot :+1:

**Comment by rafaelbn** [comment]
Why travis fails?

**Comment by dgzurita** [comment]
Tested in runbot :+1:

**Comment by yajo** [comment]
Travis happy.

File: website_crm_quick_answer/data/automation.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+<record id="website_form_leads_filter" model="ir.filters">
+    <field name="name">Coming from the website contact form</field>
+    <field name="model_id">crm.lead</field>
+    <field name="domain"
+           eval="[('section_id', '=',
```
Comment (antespi) [inline website_crm_quick_answer/data/automation.xml]: IMHO we would use here `medium_id` and `crm.crm_medium_website`

**Comment by yajo** [comment]
Please remove _needs fixing_, this seems ready to merge at my sight. @antespi's comments fixed.

**Comment by rafaelbn** [comment]
@Yajo where is runbot?

**Comment by yajo** [comment]
No idea, I guess it's being built.

**Comment by JavierIniesta** [comment]
Tested in local environment. :+1:

**Pull request merged**